run/runTPC.cxx

```diff
@@ -20,7 +20,7 @@
 #include "../macro/run_sim_tpc.C"
 #include "../macro/run_digi_tpc.C"
 #include "../macro/run_clus_tpc.C"
-#include "../Detectors/TPC/reconstruction/macro/runCATracking.C"
+//#include "../Detectors/TPC/reconstruction/macro/runCATracking.C"
```

Comment:
since the macro will be removed it does not make sense to keep the commented line, and also the commented calls should be removed